compiler+runtime/src/cpp/jank/read/parse.cpp

```diff
@@ -1196,17 +1200,18 @@ namespace jank::read::parse
         {
           ns = ns_portion;
         }
-        /* Normal symbols will have the ns resolved immediately. */
+        /* Normal symbols will have the ns resolved immediately if resolvable. */
         else
         {
           auto const resolved_ns(__rt_ctx->resolve_ns(make_box<obj::symbol>(ns_portion)));
           if(resolved_ns.is_none())
           {
-            return error::parse_unresolved_namespace(
-              fmt::format("Unresolved namespace '{}'", ns_portion),
-              { start_token.start, latest_token.end });
```

Comment:
This error is no longer used and should be removed entirely (from the enum, its uses, and this fn).